memCheck/topProcess.py

```diff
@@ -0,0 +1,26 @@
+def topProcess(numOfProcesses):
+    import psutil
+
+    ad_pids = []
+    procs = []
+    for p in psutil.process_iter():
+        with p.oneshot():
+            try:
+                mem = p.memory_full_info()
+                info = p.as_dict(attrs=['name', 'memory_info'])
+            except psutil.AccessDenied:
+                ad_pids.append(p.pid)
+            except psutil.NoSuchProcess:
+                pass
+            else:
+                p._uss = mem.uss
```

Comment:
the use of uss instead of rss would be a more accurate representation of what memory a process consumes as it represents what would be freed at that time if the pid was killed.